### PR TITLE
Ft configuration Keys

### PR DESCRIPTION
**Pull request by mubarak23**
- set TAHUB_PUBLIC_KEY and TAHUB_PRIVATE_KEY
- return TAHUB_PUBLIC_KEY on Content =" GET_SERVER_PUBKEY "

### CODE REVIEW

File: controllers_v2/nostr.ctrl.go [hunk 1]
```diff
@@ -24,53 +24,51 @@ type CreateUserEventResponseBody struct {
...
-func (controller *NostrController) HandleNostrEvent(c echo.Context) error {
+type GetTahubPublicKey struct {
+	TaHubPublicKey   string `json:"tahubpublickey"`
+	Pubkey string `json:"pubkey"`
+}
+
+
+func (controller *NostrController) AddNostrEvent(c echo.Context) error {
 	
-	var body nostr.Event
```
Comment (Nsandomeno) [inline controllers_v2/nostr.ctrl.go:29]: Lets use the event struct from this library, `go-nostr`, as their is much to take advantage from and which was pulled into the branch as a change. One immediate reason is verifying signatures.

File: controllers_v2/nostr.ctrl.go [hunk 2]
```diff
@@ -24,53 +24,51 @@ type CreateUserEventResponseBody struct {
...
 }
 
-func (controller *NostrController) HandleNostrEvent(c echo.Context) error {
+type GetTahubPublicKey struct {
+	TaHubPublicKey   string `json:"tahubpublickey"`
+	Pubkey string `json:"pubkey"`
+}
+
+
+func (controller *NostrController) AddNostrEvent(c echo.Context) error {
```
Comment (Nsandomeno) [inline controllers_v2/nostr.ctrl.go:33]: I'm open to anything for naming - but "Handle" seems more fitting than "Add". Please let me know what you think.

File: controllers_v2/nostr.ctrl.go [hunk 3]
```diff
@@ -24,53 +24,51 @@ type CreateUserEventResponseBody struct {
...
+		c.Logger().Errorf("Failed to load AddNostrEvent request body: %v", err)
 		return c.JSON(http.StatusBadRequest, responses.BadArgumentsError)
 	}
 
 	if err := c.Validate(&body); err != nil {
-		c.Logger().Errorf("Invalid Nostr Event request body: %v", err)
+		c.Logger().Errorf("Invalid AddNostrEvent request body: %v", err)
 		return c.JSON(http.StatusBadRequest, responses.BadArgumentsError)
 	}
-	// check signature
```
Comment (Nsandomeno) [inline controllers_v2/nostr.ctrl.go:40]: I don't think you've merged the branch properly. Lets make sure to check work with testing or Postman usage.

File: controllers_v2/nostr.ctrl.go (hunk 1 above)
Comment (Nsandomeno) [reply controllers_v2/nostr.ctrl.go:29]: See comments here:[ ](https://trello.com/c/gmGpvdRO)

File: controllers_v2/nostr.ctrl.go [hunk 4]
```diff
@@ -24,53 +24,51 @@ type CreateUserEventResponseBody struct {
...
 		// create user success response
 		var ResponseBody CreateUserEventResponseBody
 		ResponseBody.ID = user.ID
 		ResponseBody.Pubkey = user.Pubkey
-
 		return c.JSON(http.StatusOK, &ResponseBody)
-	} else {
+	
+	case "GET_SERVER_PUBKEY":
+		var ResponseBody GetTahubPublicKey
```
Comment (Nsandomeno) [inline controllers_v2/nostr.ctrl.go:63]: Please work this into the pattern established in` nsandomeno/TAHUB_CREATE_USER`

**Comment by Nsandomeno** [comment]
I'm not sure this has merged in the latest from `nsandomeno/TAHUB_CREATE_USER` as discussed in the comments here. If you check the "Files Changed" tab you will see what I mean.

File: controllers_v2/nostr.ctrl.go (hunk 1 above)
Comment (mubarak23) [reply controllers_v2/nostr.ctrl.go:29]: am having difficulties merging conflict from the pull i did on your repo: nsandomeno/TAHUB_CREATE_USER, am going to close this PR, create a new BRANCH from your updated branch

**Pull request closed without merging**